server/schedulers/hot_region.go

```diff
@@ -230,17 +231,24 @@ func (h *balanceHotRegionsScheduler) balanceHotWriteRegions(cluster opt.Cluster)
 	return nil
 }
 
-func calcScore(storeItems map[uint64][]*statistics.HotPeerStat, cluster opt.Cluster, kind core.ResourceKind) statistics.StoreHotRegionsStat {
+func calcScore(storeHotPeers map[uint64][]*statistics.HotPeerStat, storeBytesStat map[uint64]float64, cluster opt.Cluster, kind core.ResourceKind) statistics.StoreHotRegionsStat {
 	stats := make(statistics.StoreHotRegionsStat)
-	for storeID, items := range storeItems {
-		// HotDegree is the update times on the hot cache. If the heartbeat report
-		// the flow of the region exceeds the threshold, the scheduler will update the region in
-		// the hot cache and the hotdegree of the region will increase.
+	for storeID, items := range storeHotPeers {
+		storeHots, ok := stats[storeID]
```

Comment:
`storeHots` sounds not good enough?

Reply:
What about `hotRegions`? Or `hotPeers`?

Reply:
Both are better than `storeHots` and I prefer to unify the naming of `HotPeerStat` and `storeHotRegionsDefaultLen`

Reply:
Unified similar names.